Doc/using/configure.rst

```diff
@@ -784,7 +779,7 @@ Preprocessor flags
    headers in a nonstandard directory ``<include dir>``.
 
    Both :envvar:`CPPFLAGS` and :envvar:`LDFLAGS` need to contain the shell's
-   value for setup.py to be able to build extension modules using the
+   value to be able to build extension modules using the
    directories specified in the environment variables.
```

Comment:
I've just removed ``setup.py`` rather than replacing with e.g. ``Makefile`` as I believe CPPFLAGS and LDFLAGS are used by the windows build, though I'm not sure.